scripts/eurostat/health_determinants/Alcohol_Consumption/process.py

```diff
@@ -0,0 +1,749 @@
+# Copyright 2022 Google LLC
```

Comment:
Top level folder name needs to be renamed to alcohol_consumption

Reply:
Done the change.